modules/gitops-release-notes-1-8-2.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="gitops-release-notes-1-8-2_{context}"]
+= Release notes for {gitops-title} 1.8.2
+
+{gitops-title} 1.8.2 is now available on {product-title} 4.10, 4.11, and 4.12.
+
+[id="fixed-issues-1-8-2_{context}"]
+== Fixed issues
+
+The following issues have been resolved in the current release:
+
+* Before this update, when you configured Dex using the `.spec.dex` parameter and tried to log in to the Argo CD UI by using *Log In Via Openshift*, you were not able to log in. This update fixes the issue. 
```

Comment:
```suggestion
* Before this update, when you configured Dex using the `.spec.dex` parameter and tried to log in to the Argo CD UI by using *Log In Via Openshift*, you were not able to log in. This update fixes the issue.  link:https://issues.redhat.com/browse/GITOPS-2761[GITOPS-2761]
```
Maybe instead of adding the Jira link inside the note

Reply:
@varshab1210 thanks for the feedback. This is the standard way/template that we follow as per the RH guidelines of how to address the issues pan OCP docs for RN.

Reply:
Easier to track and search as well when we mention issue number explicitly like this.